rotkehlchen/db/filtering.py

```diff
@@ -1004,7 +1005,7 @@ def make(
             filters.append(DBMultiStringFilter(
                 and_op=True,
                 column='identifier',
-                operator='NOT IN',
+                operator=ignored_assets_operator,  # type: ignore  # ignored_assets_operator is definitely not None  # noqa: E501
```

Comment:
I think you may be able to get rid of the type ignore here if you use @overload to specify the relation that when calling with `ignored_assets_identifiers` being not None you always have an operator